aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -263,6 +264,15 @@ Tensor& addmm_out_cuda_impl(Tensor& result, const Tensor& self, const Tensor& ma
               result_ptr,
               result_ld);
         });
+    switch (activation) {
+      case cuda::blas::GEMMAndBiasActivationEpilogue::RELU:
+        at::relu_(const_cast<Tensor&>(*result_));
+        break;
+      case cuda::blas::GEMMAndBiasActivationEpilogue::GELU:
+        result_ = c10::MaybeOwned<Tensor>::owned(at::gelu(*result_));
```

Comment:
why relu_ is inplace, but gelu isn't?

Reply:
IIRC we don't have in-place gelu, but maybe I did something wrong.

Reply:
Looks like we have gelu.out and regular gelu, but no in-place.